src/components/swap/styleds.tsx

```diff
@@ -4,12 +4,28 @@ import { ReactNode } from 'react'
 import { AlertTriangle } from 'react-feather'
 import { Text } from 'rebass'
 import styled, { css } from 'styled-components/macro'
+import { Z_INDEX } from 'theme'
 
 import { AutoColumn } from '../Column'
 
-export const Wrapper = styled.div`
+export const PageWrapper = styled.div<{ redesignFlag: boolean }>`
+  padding: 0 8px;
+  max-width: ${({ redesignFlag }) => (redesignFlag ? '420px' : '480px')};
+  width: 100%;
+`
+
+// Mostly copied from `AppBody` but it was getting too hard to maintain backwards compatibility.
+export const SwapWrapper = styled.main<{ margin?: string; maxWidth?: string; redesignFlag: boolean }>`
   position: relative;
+  background: ${({ theme, redesignFlag }) => (redesignFlag ? theme.backgroundSurface : theme.deprecated_bg0)};
+  border-radius: ${({ redesignFlag }) => (redesignFlag ? '16px' : '24px')};
+  border: 1px solid ${({ theme, redesignFlag }) => (redesignFlag ? theme.backgroundOutline : 'transparent')};
   padding: 8px;
+  z-index: ${Z_INDEX.deprecated_content};
```

Comment:
do we still want to be using deprecated_content?

Reply:
yea i guess so, i just copied it over. we use deprecated everywhere else in this file for now